### PR TITLE
make deploy-example will now deploy full cluster

### PR DESCRIPTION
**Pull request by richm**
This allows you to do a single `make deploy-example` and get
a full cluster.  It assumes you have the latest
`elasticsearch-operator` cloned to `../elasticsearch-operator`

### CODE REVIEW

**Comment by richm** [comment]
with this commit and https://github.com/openshift/origin-aggregated-logging/pull/1442 we are able to run the full CI tests against a logging cluster deployed with operators

File: Makefile [hunk 1]
```diff
@@ -77,6 +78,9 @@ deploy-image:
 deploy: deploy-setup image deploy-image
 	hack/deploy.sh
 
+deploy-example: deploy
+	oc create -n $(NAMESPACE) -f hack/cr.yaml
```
Comment (ewolinetz) [inline Makefile]: `hack/cr.yaml` is missing the managed spec field, `hack/examples/cr.yaml` has it though.
I'm find with whichever one we use but we should ensure it has that spec and lets remove the dupe

**Comment by richm** [comment]
working now - please re-lgtm

**Pull request merged**